mis_builder/models/mis_report_instance.py

```diff
@@ -533,6 +557,22 @@ def _onchange_dates(self):
                     self.date_to != self.date_range_id.date_end:
                 self.date_range_id = False
 
+    @api.multi
+    def _add_analytic_filters_to_context(self, context):
+        if self.analytic_account_id:
```

Comment:
multi with a `self` usage can be dangerous

Reply:
Fixed in https://github.com/acsone/mis-builder/pull/2